plugins/periskop/src/plugin.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  discoveryApiRef,
+  configApiRef,
+  createApiFactory,
+  createPlugin,
+  createApiRef,
+} from '@backstage/core-plugin-api';
+import { PeriskopApi } from './api';
+
+import { rootRouteRef } from './routes';
+
+/**
+ * @public
+ */
+export const periskopApiRef = createApiRef<PeriskopApi>({
+  id: 'plugin.periskop.service',
+});
+
+/**
+ * @public
+ */
+export const periskopPlugin = createPlugin({
+  id: 'periskop',
+  routes: {
+    root: rootRouteRef,
```

Comment:
if you don't have a routable home page at that route, you can actually just remove the route - no harm in having it in if you plan to make some kind of overview page later